templates/etc/init.d/deb/td-agent

```diff
@@ -118,7 +118,7 @@ do_start() {
   if kill_by_file -0 "${TD_AGENT_PID_FILE}"; then
     return 1
   else
-    GOOGLE_LOGGING_ENABLE=$(curl --silent -f -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/attributes/google-logging-enable 2>/dev/null)
+    GOOGLE_LOGGING_ENABLE=$(curl --silent --connect-timeout 10 -f -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/attributes/google-logging-enable 2>/dev/null)
```

Comment:
You could make it a lot smaller, e.g., 1s.

Reply:
I don't have any insight into the performance profile of the GCP metadata server, thus my concern was that on a potentially busy client or some backend issues on GCP, long connection times are a possibility. 
I do not have a strong lean here so I'm happy to lower it if that's your inclination.

Reply:
A working metadata server should respond well within the 1s deadline (there's no actual network involved). The point here is to prevent the agent init script from waiting for a nonexistent metadata server. Let's lower it to 1s. Thanks.

Reply:
OK done!